migrations/1577932324349_add-captcha-quizid.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.addColumns("captchas", {
+        text: {
+            type: "varchar(1000)",
+            default: "",
+            notNull: true,
+        },
+        quiz_id: {
+            type: "varchar(1000)",
+            default: "",
+            notNull: true,
+        }
+    });
+    pgm.createIndex("captchas", "user_id");
+};
+
+exports.down = (pgm) => {
+    pgm.dropColumns("captchas", ["text", "quiz_id"])
+    pgm.dropIndex("captchas", "user_id");
```

Comment:
Because of the changes I made in the now merged 3 question branch this will have to include SQL which resets the table back to the original structure.

Reply:
Preferably this is done in a separate migration which precedes these new ones.

Reply:
I am using the captchas.completed field from your 3 questions branch so that does not have to be removed.